proxies/live/apiproxy/targets/poll.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I'm not sure what we decided to do with this file. Keep it as a duplication of `ig3.xml` or add logic to change `<Path>` dynamically.